infrasim/console.py

```diff
@@ -133,26 +135,29 @@ def _free_resource():
         thread.join()
 
 
-def start(instance="node-0"):
+def start(instance="default"):
     """
     Attach ipmi-console to target instance specified by
     its name
     :param instance: infrasim instance name
     """
     daemon.daemonize("{}/{}/.ipmi_console.pid".format(config.infrasim_home, instance))
-
     # initialize logging
-    common.init_logger()
+    common.init_logger(instance)
```

Comment:
Hi, @WinnieXU1 , you'd need to update the function to parse the variable `instance` as well. I believe you've planned to do so.